scipy/sparse/_matrix_io.py

```diff
@@ -94,58 +97,68 @@ def load_npz(file):
 
     See Also
     --------
-    scipy.sparse.save_npz: Save a sparse matrix to a file using ``.npz`` format.
+    scipy.sparse.save_npz: Save a sparse array/matrix to a file using ``.npz`` format.
     numpy.load: Load several arrays from a ``.npz`` archive.
 
     Examples
     --------
-    Store sparse matrix to disk, and load it again:
+    Store sparse array/matrix to disk, and load it again:
 
     >>> import numpy as np
-    >>> import scipy.sparse
-    >>> sparse_matrix = scipy.sparse.csc_matrix(np.array([[0, 0, 3], [4, 0, 0]]))
-    >>> sparse_matrix
-    <2x3 sparse matrix of type '<class 'numpy.int64'>'
+    >>> import scipy as sp
+    >>> sparse_array = sp.sparse.csc_array([[0, 0, 3], [4, 0, 0]])
+    >>> sparse_array
+    <2x3 sparse array of type '<class 'numpy.int64'>'
        with 2 stored elements in Compressed Sparse Column format>
-    >>> sparse_matrix.toarray()
+    >>> sparse_array.toarray()
     array([[0, 0, 3],
            [4, 0, 0]], dtype=int64)
 
-    >>> scipy.sparse.save_npz('/tmp/sparse_matrix.npz', sparse_matrix)
-    >>> sparse_matrix = scipy.sparse.load_npz('/tmp/sparse_matrix.npz')
+    >>> sp.sparse.save_npz('/tmp/sparse_array.npz', sparse_array)
+    >>> sparse_array = sp.sparse.load_npz('/tmp/sparse_array.npz')
 
-    >>> sparse_matrix
-    <2x3 sparse matrix of type '<class 'numpy.int64'>'
+    >>> sparse_array
+    <2x3 sparse array of type '<class 'numpy.int64'>'
         with 2 stored elements in Compressed Sparse Column format>
-    >>> sparse_matrix.toarray()
+    >>> sparse_array.toarray()
     array([[0, 0, 3],
            [4, 0, 0]], dtype=int64)
-    """
 
+    In this example we force the result to be csr_array from csr_matrix
+    >>> sparse_matrix = sp.sparse.csc_matrix([[0, 0, 3], [4, 0, 0]])
+    >>> sp.sparse.save_npz('/tmp/sparse_matrix.npz', sparse_matrix)
+    >>> tmp = sp.sparse.load_npz('/tmp/sparse_matrix.npz')
+    >>> sparse_array = sp.sparse.csr_array(tmp)
+    """
     with np.load(file, **PICKLE_KWARGS) as loaded:
         try:
-            matrix_format = loaded['format']
+            sparse_format = loaded['format']
         except KeyError as e:
             raise ValueError(f'The file {file} does not contain a sparse matrix.') from e
```

Comment:
Nothing wrong with this, but feels a bit convoluted. I know it's not yours :) How about:

```
sparse_format = loaded.get('format')
if sparse_format is None:
   ...
```